[2022-06-08 03:14:57,403] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T02:39:54.959518+00:00 [queued]>
[2022-06-08 03:14:57,424] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T02:39:54.959518+00:00 [queued]>
[2022-06-08 03:14:57,425] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 03:14:57,426] {taskinstance.py:1357} INFO - Starting attempt 2 of 3
[2022-06-08 03:14:57,427] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-08 03:14:57,448] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): install_sqoop> on 2022-06-08 02:39:54.959518+00:00
[2022-06-08 03:14:57,455] {standard_task_runner.py:52} INFO - Started process 2667 to run task
[2022-06-08 03:14:57,460] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'hands_on_test', 'install_sqoop', 'manual__2022-06-08T02:39:54.959518+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpidcwbge5', '--error-file', '/tmp/tmpijz806pp']
[2022-06-08 03:14:57,462] {standard_task_runner.py:80} INFO - Job 45: Subtask install_sqoop
[2022-06-08 03:14:57,544] {task_command.py:369} INFO - Running <TaskInstance: hands_on_test.install_sqoop manual__2022-06-08T02:39:54.959518+00:00 [running]> on host b79bd65fcc57
[2022-06-08 03:14:57,645] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=pathairs
AIRFLOW_CTX_DAG_ID=hands_on_test
AIRFLOW_CTX_TASK_ID=install_sqoop
AIRFLOW_CTX_EXECUTION_DATE=2022-06-08T02:39:54.959518+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-08T02:39:54.959518+00:00
[2022-06-08 03:14:57,647] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-08 03:14:57,648] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker exec hive-server bash /opt/sqoop/install_sqoop.sh ']
[2022-06-08 03:14:57,661] {subprocess.py:85} INFO - Output:
[2022-06-08 03:14:57,809] {subprocess.py:92} INFO -   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
[2022-06-08 03:14:57,814] {subprocess.py:92} INFO -                                  Dload  Upload   Total   Spent    Left  Speed
[2022-06-08 03:15:04,919] {subprocess.py:92} INFO -   0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0  0 17.1M    0  102k    0     0  88039      0  0:03:23  0:00:01  0:03:22 87994  8 17.1M    8 1428k    0     0   672k      0  0:00:26  0:00:02  0:00:24  671k 22 17.1M   22 3992k    0     0  1277k      0  0:00:13  0:00:03  0:00:10 1277k 36 17.1M   36 6456k    0     0  1564k      0  0:00:11  0:00:04  0:00:07 1564k 54 17.1M   54 9638k    0     0  1875k      0  0:00:09  0:00:05  0:00:04 1969k 75 17.1M   75 12.9M    0     0  2163k      0  0:00:08  0:00:06  0:00:02 2665k100 17.1M  100 17.1M    0     0  2464k      0  0:00:07  0:00:07 --:--:-- 3229k
[2022-06-08 03:15:28,057] {subprocess.py:92} INFO - mv: inter-device move failed: '/opt/sqoop/sqoop-1.4.7.bin__hadoop-2.6.0' to '/usr/lib/sqoop/sqoop-1.4.7.bin__hadoop-2.6.0'; unable to remove target: Directory not empty
[2022-06-08 03:15:32,038] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-08 03:15:32,086] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=hands_on_test, task_id=install_sqoop, execution_date=20220608T023954, start_date=20220608T031457, end_date=20220608T031532
[2022-06-08 03:15:32,149] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-08 03:15:32,215] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
